pkg/logs/input/kubernetes/launcher.go

```diff
@@ -129,7 +129,11 @@ func (l *Launcher) addSource(service *service.Service) {
 		return
 	}
 
-	l.sourcesByContainer[service.GetEntityID()] = source
+	if svc.Type == service.Containerd {
```

Comment:
why do you need to check the service type ? I think you could just call:
```
source.SetSourceType(config.ContainerdType)
```

Reply:
Because the service type could be docker, and we don't want to apply any parser in that case (because the logs are already json formatted)